control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -596,13 +600,17 @@ func (r *Reconciler) reconcileInitialOffset(ctx context.Context, cg *kafkaintern
 		return fmt.Errorf("failed to get secret for Kafka cluster auth: %w", err)
 	}
 
+	if cg.Spec.Template.Spec.Configs.Configs == nil {
+		return fmt.Errorf("no consumer config supplied, unable to get bootstrap.servers")
+	}
```

Comment:
This seems to be better suited for `Validate` so that it gets rejected from the beginning ?